File: RepoTransBench/source_projects/Python/viralogic_py-enumerable/public_tests/test_public_performance.py
from py_linq.py_linq import Enumerable

def test_public_enumerable_list_performance():
    e = Enumerable(list(range(200)))
    l = list(e)
    assert l == list(range(200))

def test_public_enumerable_tuple_performance():
    e = Enumerable(tuple(range(120)))
    l = list(e)
    assert l == list(range(120))

File: RepoTransBench/target_projects/Python/Rust/viralogic_py-enumerable/public_tests/test_public_performance.rs
// Translated from: public_tests/test_public_performance.py

use viralogic_py_enumerable::enumerable::Enumerable;

#[test]
fn test_public_enumerable_list_performance() {
    let e = Enumerable::from((0..200).collect::<Vec<_>>());
    let l: Vec<_> = e.iter().cloned().collect();
    assert_eq!(l, (0..200).collect::<Vec<_>>());
}

#[test]
fn test_public_enumerable_tuple_performance() {
    let e = Enumerable::from((0..120).collect::<Vec<_>>());
    let l: Vec<_> = e.iter().cloned().collect();
    assert_eq!(l, (0..120).collect::<Vec<_>>());
}